### DISCUSSION TITLE
false positive - malware / Trojan.Gen.MBT

### DISCUSSION THREAD

**Discussion by thezoggy** [sabnzbd/sabnzbd#2757] [General]
Just to document this here in case others have similar issue.

Win10 Pro x64 (22H2)
SAB 4.2.0rc2 windows binary, installed since the Dec 17th
This instance is not being used (spun it up to test the recent stall bug), but its just been idling for the past week.

Symantec Endpoint Protection (14.3) flagged the file a week later:
![sab](http://zoggy.net/SavUI_2023-12-24_13-09-32.png)

Which eventually resulted in it deleting sabnzbd.exe
![sab2](http://zoggy.net/SymCorpUI_2023-12-25_01-33-29.png)

What is odd, is that there was an exception from ages ago that should have even prevented this from being flagged/touched:
![symantec](http://zoggy.net/SymCorpUI_2023-12-25_01-35-52.png)

Submitted file for review at Symantec's site and linked to this discussion.
> False Positive submission (351674)

**Comment by thezoggy** [comment]
Symantec got back and removed 

> In relation to submission 351674.
> 
> Upon further analysis and investigation we have verified your submission and, as such, the detection(s) for the following file(s) will be removed from our products:
> 
>    File name: SABnzbd.exe
>    MD5: 8FD049783B94D2921A58185FE3F5C780
>    SHA256: 5A26D4EAEA8CFEDDAF1CE2F09A3B9231F337320B910949126B8AA73FAFD5E8D0
>    Note: Whitelisting is available by downloading a RAPID RELEASE indicated in the Further Information section below or via the next Live Update
> Further Information:
> Required RAPID RELEASE sequence >= 232242

**Reply by Safihre** [reply]
👍
